index.html

```diff
@@ -139,55 +139,76 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                            <li class="skill-tag">CSS</li>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div class="skills">
-                <img src="assets/Frameworks.svg" alt="Frameworks">
-                <div>
-                    <h2 class="skill-title">Frameworks</h2>
-                    <ul>
-                        <li class="skill-tag">React.js</li>
-                        <li class="skill-tag">Ruby on Rails</li>
-                        <li class="skill-tag">RSpec</li>
-                        <li class="skill-tag">Capybara</li>
-                        <li class="skill-tag">Selenium</li>
-                    </ul>
+                <div class="skills">
+                    <img src="assets/Frameworks.svg" alt="Frameworks">
+                    <div>
+                        <h2 class="skill-title color1">Frameworks</h2>
+                        <ul>
+                            <li class="skill-tag">React.js</li>
+                            <li class="skill-tag">Ruby on Rails</li>
+                            <li class="skill-tag">RSpec</li>
+                            <li class="skill-tag">Capybara</li>
+                            <li class="skill-tag">Selenium</li>
+                        </ul>
+                    </div>
                 </div>
-            </div>
-            <div class="skills">
-                <img src="assets/Skills.svg" alt="Languages">
-                <div>
-                    <h2 class="skill-title">Skills</h2>
-                    <ul>
-                        <li class="skill-tag">Database Management</li>
-                        <li class="skill-tag">Version Control</li>
-                        <li class="skill-tag">CLI</li>
-                        <li class="skill-tag">Web Development</li>
-                        <li class="skill-tag">API Design</li>
-                    </ul>
+                <div class="skills">
+                    <img src="assets/Skills.svg" alt="Languages">
+                    <div>
+                        <h2 class="skill-title color1">Skills</h2>
+                        <ul>
+                            <li class="skill-tag">Database Management</li>
+                            <li class="skill-tag">Version Control</li>
+                            <li class="skill-tag">CLI</li>
+                            <li class="skill-tag">Web Development</li>
+                            <li class="skill-tag">API Design</li>
+                        </ul>
+                    </div>
                 </div>
             </div>
-        </div>
+        </section>
+        <section class="contact">
+                <h2 class="primary color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="" class="form" action="https://formspree.io/f/mwkjgvpr"
+            method="POST">
```

Comment:
- [ ] Kindly note that because you are providing two `action` attributes, your data are not collected, thus we get this error page, kindly remove the empty `action` attribute.
```suggestion
            <form class="form" action="https://formspree.io/f/mwkjgvpr"
            method="POST">
```
  
 ![image](https://github.com/BHObserver/portfolio/assets/102612396/3a3908b6-a063-4197-b85c-fcb64638a033)